ui/src/features/components/TestForm/utils.js

```diff
@@ -9,7 +9,7 @@ import {
 
 const SLEEP = 'sleep';
 export const createTestRequest = (data) => {
-    const {name, description, scenarios, type, baseUrl, before, processorId, csvFileId} = data;
+    const {name, description, scenarios, type, baseUrl, before, processorId, csvFileId, isFavorite} = data;
```

Comment:
@manorlh will review the UI part
but one thing that can be improved is to query to test with filter query param instead of fetching all tests and then filtering in the UI side